src/transformers/convert_graph_to_onnx.py

```diff
@@ -214,25 +320,60 @@ def convert(
         convert_tensorflow(nlp, opset, output)
 
 
-def verify(path: str):
+def quantize(onnx_model_path: Path) -> Path:
```

Comment:
Thanks Morgan for integrating onnx quantization! This is the overall process of onnx quantization. For transformer-based models, it gets better performance by optimizing with onnxruntime_tools.optimizer before quantizing. Below is an example to optimize Bert-base with the tool:

# optimize transformer-based models with onnxruntime-tools
from onnxruntime_tools import optimizer
from onnxruntime_tools.transformers.onnx_model_bert import BertOptimizationOptions

# disable embedding layer norm optimization for better model size reduction
opt_options = BertOptimizationOptions('bert')
opt_options.enable_embed_layer_norm = False

opt_model = optimizer.optimize_model(
    'bert.onnx',
    'bert', 
    num_heads=12,
    hidden_size=768,
    optimization_options=opt_options)
opt_model.save_model_to_file('bert.opt.onnx')

onnx_opt_model = onnx.load('bert.opt.onnx')
quantized_onnx_model = quantize(onnx_opt_model,
                                    quantization_mode=QuantizationMode.IntegerOps,
                                    symmetric_weight=True,
                                    force_fusions=True)

onnx.save(quantized_onnx_model, 'bert.opt.quant.onnx')